go/atreugo/go.mod

```diff
@@ -0,0 +1,5 @@
+module main
+
+go 1.12
+
+require github.com/savsgio/atreugo/v8 v8.1.1
```

Comment:
@savsgio @kirillDanshin @jwilner is there a way in `go` to use `go` **module** with `X.Y` version of a dependency ?

I mean can we write `require github.com/savsgio/atreugo/v8 v8.1.*` to use any version between 8.1 and (excluded) 8.2 ?

Reply:
You could see: https://github.com/golang/go/wiki/Modules#version-selection.
But It's mandatory to set `X.Y.Z`